include/ensmallen_bits/nsga2/nsga2_impl.hpp

```diff
@@ -113,6 +114,7 @@ typename MatType::elem_type NSGA2::Optimize(
 
   // Generate the population based on a uniform distribution around the given
   // starting point.
+  //TODO: Shouldn't we check if they're inside the upperBound && lowerBound?
   for (size_t i = 0; i < populationSize; i++)
   {
     population.push_back(arma::randu<MatType>(iterate.n_rows,
```

Comment:
Shouldn't we check if they're inside the upperBound && lowerBound? Am I missing something? 

An example https://github.com/baopng/NSGA-II/blob/29a6ec33f87b32a7fb2596091e1a51c897106e7b/nsga2/problem.py#L20